src/main/java/org/spongepowered/api/event/message/CommandEvent.java

```diff
@@ -61,4 +65,19 @@
      */
     String getArguments();
 
+    /**
+     * The result of the command. This is only available after the execution 
+     * of the command.
+     * 
+     * @return The result of the command, if present
+     */
+    Optional<CommandResult> getResult();
+
+    /**
+     * Sets the result of the command.
+     * 
+     * @param result The result of the command, or null
+     */
+    void setResult(@Nullable CommandResult result);
```

Comment:
You haven't added the key 'result' to the [`SpongeEventFactory`](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/event/SpongeEventFactory.java#L909)